frontend/talentsearch/src/js/components/Browse/OngoingRecruitmentSection.tsx

```diff
@@ -0,0 +1,1191 @@
+          description:
+            "instructions for section with ongoing pool advertisements",
+        })}
+      </p>
+      <div data-h2-padding="base(x2, 0, 0, 0) p-tablet(x3, 0, 0, 0)">
+        <Accordion.Root
+          data-h2-margin="base(0)"
+          data-h2-padding="base(0)"
+          type="multiple"
+        >
+          {dataModel.streams
+            .filter(
+              // filter streams with no classifications with a pool ID to apply to
+              (stream) =>
+                !!stream.classifications.find(
+                  (classification) => classification.poolAdvertisementId,
+                ),
+            )
+            .map((stream) => (
+              <Accordion.Item value={stream.key} key={stream.key}>
+                <Accordion.Trigger
+                  subtitle={stream.summary}
+                  data-h2-font-size="base(h4)"
+                >
+                  {stream.title}
+                </Accordion.Trigger>
+                <Accordion.Content>
+                  <div
+                    data-h2-display="base(grid)"
+                    data-h2-grid-template-columns="base(1fr) p-tablet(1fr 1fr)"
+                    data-h2-gap="base(x.5) p-tablet(x2)"
+                    data-h2-align-items="base(center)"
+                  >
+                    {stream.classifications
+                      .filter(
+                        // filter to only classifications with a pool ID that can be applied to
+                        (classification) => classification.poolAdvertisementId,
+                      )
+                      .map((classification) => (
+                        <div
+                          key={classification.title}
+                          data-h2-padding="base(0,0,x1, 0)"
+                        >
+                          <h3
+                            data-h2-font-size="base(h6)"
+                            data-h2-font-weight="base(700)"
+                            data-h2-padding="base(0,0,x0.75, 0)"
+                          >
+                            {classification.title}
+                          </h3>
+                          <p data-h2-padding="base(0,0,x0.75, 0)">
+                            {classification.description}
+                          </p>
+                          {classification.poolAdvertisementId && (
+                            <Link
+                              href={paths.pool(
+                                classification.poolAdvertisementId,
+                              )}
+                              color="blue"
+                              type="button"
+                              mode="solid"
+                              data-h2-font-weight="base(700)"
+                            >
+                              {classification.applyMessage}
```

Comment:
I know we are trying to move away from the `<hidden />` tag due to the bug in VoiceOver but I do think this link could use a bit more context to prevent having multiple links saying "Apply to level X" on the page.

Perhaps something like that results in "Apply to {stream.title} level X"

Reply:
OK, I've added the stream name just tacked onto the end so as not to tangle with translation.  I've also factored out the bulk of the messages to a separate message file since it was just too much copy/paste.
Changed in [update translation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5043/commits/0de44af5bf714c0a65a188dc9de7b697d72371fe)